flux/dev/config.yaml

```diff
@@ -0,0 +1,21 @@
+external_package_dirs: []
+model_metadata:
+  example_model_input: {"prompt": 'black forest gateau cake spelling out the words "FLUX DEV", tasty, food photography, dynamic shot'}
```

Comment:
thanks for adding this

Reply:
👍 I think its in the CONTRIBUTING.md as a requirement lol